pkg/cluster/manifests/values.yaml

```diff
@@ -12,9 +12,14 @@ global:
   masterNodeCount: 1
   nodeType: t3.large
   kubeconfigSecretKeyName: value
+  clusterAutoscalerEnabled: false
+  clusterAutoscalerMinNodes: "1"
+  clusterAutoscalerMaxNodes: "9"
```

Comment:
Maybe it is my inexperience with Arlon, but aren't there many more configuration parameters related to CAS? Will we be adding those here as well? If not, where do we draw the line?